sccp/imports/scRNA.py

```diff
@@ -216,7 +216,8 @@ def ThompsonXA_SCGenes(saveXA=False, offset=1.0):
             celltypeXA = xa.open_dataarray("/opt/andrew/scRNA_celltypeXA_NoOffset.nc")
         else:
             XA = xa.open_dataarray("/opt/andrew/scRNA_drugXA.nc")
-
+            
+    XA.values -= np.nanmean(XA.values, axis=(0,1), keepdims=True)
```

Comment:
@aarmey For some reason this transformation in the code creates many empty projections and cell types later in the code and greatly changes the results. 

I changed the normalization to  `XA.values /= np.nanmean(XA.values, axis=(0,1), keepdims=True)` and results dont create the same issue. 
<img width="327" alt="Screen Shot 2023-03-16 at 2 13 34 PM" src="https://user-images.githubusercontent.com/91349614/225754252-a00c6805-d34e-499f-891c-39efbb91a55e.png">
<img width="320" alt="Screen Shot 2023-03-16 at 2 13 46 PM" src="https://user-images.githubusercontent.com/91349614/225754255-dab86f07-8e84-4a68-9afa-52a8b6b59d7a.png">

Reply:
Aha! So this has an issue I was wondering about last night—this is being performed after we fill in the nans with 0. So, we need to do this earlier in the initial data processing.